text-utility/text-utility.css

```diff
@@ -0,0 +1,62 @@
+.head {
+  color: #000;
```

Comment:
yes, you can define font-weight here for headings.

Reply:
Done.. For heading i have added the font-weight separately..

Reply:
okay